src/app/store/slices/user.ts

```diff
@@ -20,6 +22,11 @@ interface AuthAction {
   isAuth: UserState["isAuth"];
 }
 
+// add actions for role recognice
+interface RoleAction {
+  role: UserState["role"];
```

Comment:
Create Union type for roles and use it here

Reply:
complete